cockroachcloud/authorization.md

```diff
@@ -1,44 +1,158 @@
+
+See: [Managing Access in CockroachDB Cloud: Manage organization users](managing-access.html#manage-an-organizations-users)
+
+The following roles may be granted to {{ site.data.products.db }} organization users within a specific organization:
+
+- Org member
+- Org Administrator (legacy)
+- Org developer (legacy)
+- Cluster developer
+- Cluster admin
+- Cluster creator
+
+### Organization member
+The default role given to all organization users upon creation or invitation. This role grants no permissions to perform cluster or org actions.
+### Org Administrator (legacy)
+The administrator role for organization functions. This role grants the user permissions to perform all critical functions managing a {{ site.data.products.db }} organization:
+
+- [Create a cluster](create-your-cluster.html)
+- [Invite Team Members to the organization](managing-access.html#invite-team-members-to-an-organization)
+- [Manage an organization's users and their roles](managing-access.html#manage-an-organizations-users)
+- [Create and manage SQL users](managing-access.html#create-a-sql-user)
+- [Manage billing for the organization](billing-management.html)
+- [Restore databases and tables from a {{ site.data.products.db }} backup](use-managed-service-backups.html#ways-to-restore-data)
+- [Delete an organization](managing-access.html#delete-an-organization)
+{{site.data.alerts.callout_info}}
+In a future release, this role will be deprecated in favor of more fine-grained roles for separately administrating organization-level user-management functions, cluster functions, and billing functions.
+{{site.data.alerts.end}}
+  
+### Org developer (legacy)
+This role allows the user to read information for all clusters, and to create and manage SQL users on all clusters. It is considered deprecated in favor of the more fine-grained cluster roles below.
+### Cluster developer
+The minimum access role for clusters. This role allows users to view the details of one or more specified clusters, as well as change their IP allowlist configuration.
+### Cluster administrator
+The administrator role for a specific database cluster, including managing SQL users/roles. This role can be granted to an organization user for one or more specific clusters, or for all clusters in the organization.
+### Cluster creator
+This role alows an organization user to create clusters in an organization; the cluster creator is automatically granted the cluster admin role on clusters they create.
 
-Users may be granted two levels of access:
+## Service accounts
 
-- 'Developer' allows users to:
-  - Read cluster information using the CockroachDB Cloud Console and Cloud API
-- 'Administrator' allows users to:
-  - Create and delete organization users
-  - Create, modify, and delete clusters
-  - Create and manage SQL roles on particular clusters
+Service accounts differ from users in that they authenticate with API keys, rather than through the {{ site.data.products.db }} console.
```

Comment:
```suggestion
Service accounts differ from users in that they authenticate with API keys, rather than through the {{ site.data.products.db }} console UI.
```